[DOCS] Rework conceptual info for ILM. (#52181)
elastic/elasticsearch: docs/reference/ilm/actions/ilm-freeze.asciidoc

[[ilm-freeze]]
=== Freeze

Phases allowed: cold.

> This action will <<frozen-indices, freeze>> the index
> by calling the <<freeze-index-api, Freeze Index API>>.

Reviewer: ```suggestion
<<frozen-indices, freezes>> an index
using the <<freeze-index-api, Freeze Index API>>.
```

> [source,console]

Reviewer: Add `Examples` heading
